ImgShareData/ImgurApiSource.cs

```diff
@@ -177,12 +177,181 @@ protected internal void EnsureClientHasAuthorizationHeaders()
         /// Get information about a specific album
         /// </summary>
         /// <param name="albumID">The requested album ID</param>
-        /// <returns></returns>
+        /// <returns>A filled in album object</returns>
         public async Task<ImgurAlbum> AlbumDetailsAsync(string albumID)
         {
             string responseString = await GetAnonymousImgurDataAsync(ImgurEndpoints.Album(albumID));
-            return await Task.Run(() => JsonConvert.DeserializeObject<ImgurAlbum>(responseString, _defaultSerializerSettings));
+            ImgurAlbumResponse x = await Task.Run(() => JsonConvert.DeserializeObject<ImgurAlbumResponse>(responseString, _defaultSerializerSettings));

```

Comment:
nit: this is a common pattern used below and whatnot - so i won't comment there and we'll resolve this here